### ISSUE TITLE
hash key expired

### ISSUE THREAD

**Issue by blake364** [pokesniperandroid/PokeSniper-Android#267]
it said hash key has expired and it wont snipe. how do i fix this??

**Comment by Jonnh117** [comment]
The same thing happens to me, I don't think it's an individual problem, but it's about updating the api of pokémon go :| I hope beta22 comes out soon

**Comment by RpapafoxR** [comment]
Same here. developers knows the issue?

**Comment by Jonnh117** [comment]
I don't think so, I hope the update comes out as soon as possible ... Star dust and x2 experience is already close

**Comment by Jonnh117** [comment]
Do you know how to contact the developer?

**Comment by Jonnh117** [comment]
I just sent him an email, I hope he answers me

**Comment by Jonnh117** [comment]
I already answered, he mentioned that they are doing their best to upgrade and since the previous version 0.73 has a strong security

**Comment by RpapafoxR** [comment]
👍

**Comment by RpapafoxR** [comment]
let's hope for the best

**Comment by Jonnh117** [comment]
Beta21 is already working again, but it is important that you use it with caution since you are at risk of

**Comment by RpapafoxR** [comment]
Do you mean risk of what? to be banned? I know, I am very careful of that, I wait at least 3 hours to snipe.

**Comment by Jonnh117** [comment]
Yes, risk of ban
